tools/streamer_recorder/include/PracticalSocket.cpp

```diff
@@ -0,0 +1,381 @@
+/*
+ *   C++ sockets on Unix and Windows
+ *   Copyright (C) 2002 Chen Xiao Qino
```

Comment:
I'm sure "Chen Xiao Qino" is not the author of this code. He is merely redistributing it.

Reply:
I've address this part.

Reply:
DONE